loky/backend/spawn.py

```diff
@@ -33,8 +34,9 @@ def get_executable():
 
 
 def _check_not_importing_main():
-    if getattr(process.current_process(), '_inheriting', False):
-        raise RuntimeError('''
+    if getattr(process.current_process(), "_inheriting", False):
+        raise RuntimeError(
```

Comment:
Maybe we could wrap this block with: `textwrap.dedent` as we typically do in for expected pytest exception messages.